core/jazz_metrics/test/test.js

```diff
@@ -581,6 +581,18 @@ describe('jazz_metrics', function () {
           "statistics": "Average"
         }
       }
+      accessparams = {
+        "accessKeyId": "hewf73223e2h3u23y82332h23nd",
+        "secretAccessKey": "48584ji4rnfruihv849fjne",
+        "sessionToken": "nckjvrevr4343k4jioj43u94j3m4oi493f43jfm43fcn",
+      };
+      serviceData = {
+        "count": 1,
+        "services" :{
+          "region" : "us-east-1",
+          "iamRoleARN": "arn:aws:iam::102707241671:role/config21_basic_execution",
```

Comment:
remove identifying information

Reply:
This is security violation, please squash the commit